example/index.js

```diff
@@ -0,0 +1,127 @@
+import { createLightNode } from "js-waku/lib/create_waku";
+import { waitForRemotePeer } from "js-waku/lib/wait_for_remote_peer";
+import * as noise from "@waku/js-noise";
```

Comment:
```suggestion
import * as noise from "@waku/noise";
```

can we change package name?